### PR TITLE
Refresh dotnet topic

### PR DESCRIPTION
**Pull request by NextTurn**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

### CODE REVIEW

File: topics/dotnet/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ logo: dotnet.png
 released: February 13, 2002
 short_description: ".NET is a software framework developed by Microsoft."
 topic: dotnet
-url: https://www.microsoft.com/net/
+url: https://dotnet.microsoft.com/
 wikipedia_url: https://en.wikipedia.org/wiki/.NET_Framework
```
Comment (TINKTWICE92) [inline topics/dotnet/index.md]: Fix

File: topics/dotnet/index.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ display_name: ".NET"
 github_url: https://github.com/dotnet
 logo: dotnet.png
 released: February 13, 2002
-short_description: ".NET is a software framework developed by Microsoft."
+short_description: ".NET is a free, cross-platform, open source developer platform."
```
Comment (leecow) [inline topics/dotnet/index.md]: Need disambiguation. Only .NET Core is cross plat and open source. .NET Framework is neither.

File: topics/dotnet/index.md [hunk 3]
```diff
@@ -5,9 +5,9 @@ display_name: ".NET"
...
 topic: dotnet
-url: https://www.microsoft.com/net/
+url: https://dotnet.microsoft.com
 wikipedia_url: https://en.wikipedia.org/wiki/.NET_Framework
 ---
-.NET is a software framework developed by Microsoft in February of 2002 as an infrastructure for building and running applications and services. The purpose of .NET is to make it easier for users to share information between websites. The framework is intended to be utilized for Windows platforms.
+.NET is a free, cross-platform, open source developer platform for building many different types of applications.
```
Comment (leecow) [inline topics/dotnet/index.md]: Same as comment above. Need to distinguish between .NET Core and .NET Framework

File: topics/dotnet/index.md (hunk 2 above)
Comment (NextTurn) [reply topics/dotnet/index.md]: There is usually one ".NET" in the [introduction](https://dotnet.microsoft.com). .NET Core, .NET Framework, Xamarin and Mono are implementations of .NET.

Comment (NextTurn) [reply topics/dotnet/index.md]: What are your thoughts @leecow?

File: topics/dotnet/index.md (hunk 3 above)
Comment (leecow) [reply topics/dotnet/index.md]: That matches the description on https://dotnet.microsoft.com so, yay for that :-) 👍

**Comment by vanluan86** [comment]
vanluan86

**Comment by vanluan86** [comment]
netgame92

File: topics/dotnet/index.md (hunk 3 above)
Comment (vanluan86) [inline topics/dotnet/index.md]: hiểu

**Comment by NextTurn** [comment]
So this is ready for merge @kenyonj

**Comment by talsafran** [comment]
Hi @NextTurn  👋 ! This looks great, could you please resolve the conflicts with the new master? Thanks!

**Pull request closed without merging**